### ISSUE TITLE
Final Peer Review

### ISSUE THREAD

**Issue by kel89** [kgcornell/Airbnb-Price-and-Occupancy-Rate-Prediction-in-Seattle#35]
I would like to start by saying that I enjoyed reading this report. From the start, it was very aesthetically pleasing (this is meaningless in terms of quality of analysis, but I enjoyed it nevertheless). I think you have an interesting and worthwhile topic--the "sharing economy" is growing rapidly, and as more people being to take part they will require tools like this to maximize the benefit from it. I like the optimization aspect of the project as well; it shows how the predictive tools can be used. 

Moving on to your data. This seems like quite a rich set, so I am surprised they do not include occupancy, but I think that proxying for it with number of reviews is both clever and likely accurate. In terms of your visualizations, it may have been worth truncating prices, as to see the one observation at 10,000o we lose all granularity with the majority of prices below 2,000. While I think the bubble plot of latitude's and longitudes is quite pretty, I do not see it's value? You did not mention after if you used this data in the models, and if so how. We talked about in class how using latitude and longitudes is difficult if included raw (correlation moving east doesn't make much sense).

I like that you start with a benchmark in the regression section, but you do not really explain what that benchmark is? You say you take the mean of the output space (so that is occupancy) and compare that with the RMSE (residual mean squared error). However, these do not have the same units, one is squared and the other is not; I struggle to see that value in this comparison?

I liked that you tried a bunch of different feature combinations to get the best fitting linear model. I am a bit concerned however that you are checking all of these on the test set? In later models you use cross validation, and I think that would have been nice to do here as well. As it stands I do not true the best linear model you found, as that could be overfitting to the test set (as you tried so many on the test set).

I think the ridge regression is a good place to start, and I like that you thought about the cross validation for lambda. What I am concerned about is the negative MSE: that is not the convention, so the reasoning behind your choice for showing it that way would be interesting.

I really enjoyed reading about the different ensemble methods you used. I have never heard of XGBoost of AdaBoost before. I have heard of random forest, and what you said worried me. There are a number of parameters you can tune in a random forest: the number of features selected at each split, the number of trees in the forest, and as you said the size of the data subset to train on. However, you do not mention at all how (or if) you attempted to tune these. I only notice on the random forest because I know those, but it worries me that you may not have done that on the XGBoost of Ada (if they have similar tuning parameters). 

I thought your optimization was quite interesting. The simulation seems like a pretty good way to do it, but the sample of only 8 listings seems a bit small? Your scope for further work seems very appropriate as well--I agree that some NLC would be a great addition.

Overall, I really enjoyed the project. I thought you used appropriate methods and your results are interesting.
